### PR TITLE
Add COSCUP transcript and images

### PR DESCRIPTION
**Pull request by oskarth**
Transcript of talk given at COSCUP in Chinese.

Already written and presented, so not looking for feedback on content. Only obvious mistakes or inaccuracies. Want to push it live today :)

### CODE REVIEW

File: _posts/2021-08-06-coscup-waku-ethereum.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+`/toychat/2/huilong/proto`
+
+For more on topics, see https://rfc.vac.dev/spec/23/
+
+### Status app
+
+In the Status protocol, content topics - topics in Whisper/Waku v1 - are used for several things:
+
+- Contact code topic to discover X3DH bundles for perfect forward secrecy
+	- Partitioned into N (currently 5000) content topics to balance privacy with efficancy
```
Comment (D4nte) [inline _posts/2021-08-06-coscup-waku-ethereum.md:171]:
```suggestion
	- Partitioned into N (currently 5000) content topics to balance privacy with efficacy
```

**Review by D4nte — APPROVED** [review]

File: _posts/2021-08-06-coscup-waku-ethereum.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+JSON RPC API, zkSNARKS based spam protection with RLN, accounting and
+settlements with SWAP, fault-tolerant store nodes, recommendations around topic
+usage, and more.
+
+See https://rfc.vac.dev/ for a full overview.
+
+### Implementations
+
+Waku v2 consists of multiple implementations. This allows for client diversity,
+makes it easier to strength the protocols, and allow people to use Waku v2 in
```
Comment (jm-clius) [inline _posts/2021-08-06-coscup-waku-ethereum.md:106]:
```suggestion
makes it easier to strengthen the protocols, and allow people to use Waku v2 in
```

File: _posts/2021-08-06-coscup-waku-ethereum.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+### Prelude: Topics in Waku v2
+
+To give some context, there are two different types of topics in Waku v2. One is
+a PubSub topic, for routing. The other is a content topic, which is used for
+content based filtering. Here's an example of the default PubSub topic:
+
+`/waku/2/default-waku/proto`
+
+This is recommended as it increase privacy for participants and it is stored by
```
Comment (jm-clius) [inline _posts/2021-08-06-coscup-waku-ethereum.md:156]:
```suggestion
This is recommended as it increases privacy for participants and it is stored by
```

**Review by jm-clius — APPROVED** [review]
Great!

File: _posts/2021-08-06-coscup-waku-ethereum.md (hunk 1 above)
Comment (oskarth) [reply _posts/2021-08-06-coscup-waku-ethereum.md:171]: this should actually be efficiency

**Comment by oskarth** [comment]
Thanks for comments both!

**Pull request merged**